app/views/cclow/home/_latest_additions.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="slider-container">
+  <div class="slider" data-slick='{"slidesToShow": 3, "slidesToScroll": 3}'>
+    <% @latest_additions.each do |item| %>
+      <div class="item-container">
+        <div class="item">
+          <div class="additions-kind"><%= item[:kind] %></div>
+          <div class="title"><a href="#"><%= item[:title] %></a></div>
```

Comment:
When clicking on the title it should lead us either to the legislation show page or the litigation show page. Could you hook that up in this PR?

Reply:
yes, of course, I will do it now

Reply:
Just an aside: it is valid to use anchors (<a></a>) without href. Having the `#` with no proper `prevendDefault()` behaviour may cause undesirable flickers.
Or there's always the `<button class="link">` that i think it will look the same, but i'd stick to anchors with no href.